OpenRA.Mods.Common/Scripting/Properties/ProductionProperties.cs

```diff
@@ -90,7 +90,7 @@ public class ProductionQueueProperties : ScriptActorProperties, Requires<Product
 		public ProductionQueueProperties(ScriptContext context, Actor self)
 			: base(context, self)
 		{
-			queues = self.TraitsImplementing<ProductionQueue>().Where(q => q.Enabled).ToArray();
+			queues = self.TraitsImplementing<ProductionQueue>().Where(q => q.UsableByFaction).ToArray();
```

Comment:
Given this, would it make sense to replace all outside uses of `UsableByFaction` with `ProductionAvailable` or `IsTraitDisabled` and simplify the external interface of the queue?